in-toto-spec.md

```diff
@@ -1158,13 +1245,18 @@ metadata:
       "stdin": "",
       "stdout": "foo.py",
       "return-value": "0"
+    },
+    "environment": {
+      "variables": [""],
```

Comment:
why variables has an empty list-like description [""], whereas filesystem and workdir have empty non-list-like description "" ?

Reply:
In the case of variables I *know* that it's going to be a list of them, on the other case, given that it's an opaque field, it could contain just a string (with e.g., the hash of the filesystem tree). I also think it makes sense to have the CWD just as a string.

Do you think I should work on clarifying this on the spec or was this more of a general question?

Reply:
I think that the general description of the environment information (Section 4.4.1) already provides clarity that "variables" is a list, "filesystem" is a list or an item, and "workdir" is just one string (one path).

(BTW, in Sec. 4.4.1, change "Variables" to "variables" -- uppercase V to lowercase v).

However, the examples in Sections 5.3.1, 5.3.2, 5.3.3 contain specific instantiations of the above general rule. As such, in these examples the "variables" field is just an empty list.
It looks strange to denote 3 fields which are empty using different notation (even if one field is a list in this case).

But, if you think this is fine, I don't feel strongly about this.

Reply:
Ah, I see. I'll check the examples and make sure they are compliant with the specification.